### PR TITLE
Possible fix for task.summary()

### PR DESCRIPTION
**Pull request by mattdangerw**
See https://github.com/keras-team/keras-nlp/issues/898.

[demo colab](https://colab.research.google.com/gist/mattdangerw/0b24ef4548e86c2872e9c21bd4d106ea/task-summaries.ipynb)

### CODE REVIEW

**Comment by mattdangerw** [comment]
Talk with @fchollet, think we should go for something like this (almost any fix will be a positive delta).

Added some tests and marking this ready for review!

File: keras_nlp/utils/keras_utils.py [hunk 1]
```diff
@@ -111,3 +113,60 @@ def is_xla_compatible(model):
             tf.distribute.TPUStrategy,
         ),
     )
+
+
+def print_msg(message, line_break=True):
```
Comment (fchollet) [inline keras_nlp/utils/keras_utils.py:118]: I wonder if we should export these 2 to __internal__ so our other packages can use them?

**Review by fchollet — APPROVED** [review]
LGTM

File: keras_nlp/utils/keras_utils.py (hunk 1 above)
Comment (mattdangerw) [reply keras_nlp/utils/keras_utils.py:118]: I think we would probably still need to copy for now, as we wouldn't want the dep on tf 2.13 anytime soon.

But could be worth thinking about this yeah. Is custom summaries a need for CV or any other keras ecosystem project?

**Pull request merged**